components/clearbit/actions/domain-lookup/domain-lookup.mjs

```diff
@@ -1,53 +1,65 @@
-// legacy_hash_id: a_xqiqma
-import { axios } from "@pipedream/platform";
+import app from "../../clearbit.app.mjs";
 
 export default {
   key: "clearbit-domain-lookup",
   name: "Domain lookup",
-  description: "The Company API allows you to look up a company by their domain",
-  version: "0.1.1",
+  description: "The Company API allows you to look up a company by their domain. [See the docs here](https://dashboard.clearbit.com/docs?javascript#enrichment-api-company-api-domain-lookup).",
+  version: "0.1.2",
   type: "action",
   props: {
-    clearbit: {
-      type: "app",
-      app: "clearbit",
-    },
+    app,
     domain: {
-      type: "string",
-      description: "The domain to look up.",
+      propDefinition: [
+        app,
+        "domain",
+      ],
     },
-    webhook_url: {
-      type: "string",
-      description: "A webhook URL that results will be sent to.",
-      optional: true,
+    webhookUrl: {
+      propDefinition: [
+        app,
+        "webhookUrl",
+      ],
     },
-    company_name: {
-      type: "string",
-      description: "The name of the company.",
+    companyName: {
+      propDefinition: [
+        app,
+        "companyName",
+      ],
       optional: true,
     },
     linkedin: {
-      type: "string",
-      description: "The LinkedIn URL for the company.",
-      optional: true,
+      propDefinition: [
+        app,
+        "linkedin",
+      ],
     },
     twitter: {
-      type: "string",
-      description: "The Twitter handle for the company.",
-      optional: true,
+      propDefinition: [
+        app,
+        "twitter",
+      ],
     },
     facebook: {
-      type: "string",
-      description: "The Facebook URL for the company.",
-      optional: true,
+      propDefinition: [
+        app,
+        "facebook",
+      ],
     },
   },
   async run({ $ }) {
-    return await axios($, {
-      url: `https://company.clearbit.com/v2/companies/find?domain=${this.domain}&webhook_url=${this.webhook_url}&company_name=${this.company_name}&linkedin=${this.linkedin}&twitter=${this.twitter}&facebook=${this.facebook}`,
-      headers: {
-        Authorization: `Bearer ${this.clearbit.$auth.api_key}`,
-      },
+    const res = await this.app.domainLookup($, {
+      domain: this.domain,
+      webhook_url: this.webhookUrl,
+      company_name: this.companyName,
+      linkedin: this.linkedin,
+      twitter: this.twitter,
+      facebook: this.facebook,
     });
+    if (res.error?.message) {
+      $.export("$summary", res.error.message);
+      throw new Error(res.error?.message);
+    }
```

Comment:
The API doesn't throw an error if there is one?
Also, I think that we don't `export summary` on errors